docs/files.md

```diff
@@ -592,7 +592,7 @@ Location: http://cozy.example.com/files/9152d568-7e7c-11e6-a377-37cbfb190b4b
 
 ### POST /files/archive
 
-Create an archive and download it. The body of the request lists the files and
+Create an archive. The body of the request lists the files and
```

Comment:
Would it be difficult to have content negotiation on that endpoint? When the client asks for `application/vnd.api+json` (JSON-API), the stack gives it a link for downloading the archive. But if it asks anything else, the stack sends the archive directly. I think it can be easier for apps that are not in a browser to have the possibility to download the archive in one request and not two.